docs/configuration.md

```diff
@@ -0,0 +1,89 @@
+# Configuration
+
+Configuring a repo to use `gha-scala-library-release-workflow` requires a `release.yml` GitHub workflow in your repo,
+and updated `sbt` settings.
+
+[Example GitHub pull requests](#examples) making these changes can be found further below.
+
+## GitHub workflow
+
+[Example `.github/workflows/release.yml`](https://github.com/guardian/etag-caching/blob/main/.github/workflows/release.yml)
+
+The functionality of `gha-scala-library-release-workflow` is provided in a
+[reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows)
+called [`reusable-release.yml`](https://github.com/guardian/gha-scala-library-release-workflow/blob/main/.github/workflows/reusable-release.yml) -
+don't copy-and-paste that big file, instead just make a small `release.yml` workflow to
+[_call_](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow)
+it (as in the example file above).
+
+Your repo will require access to [release credentials](credentials.md) to
+[pass on those secrets](https://github.com/guardian/etag-caching/blob/9935da29e76b8b89759bcfe967cc7c1c02aa1814/.github/workflows/release.yml#L11-L13)
+to the workflow.
+
+## `sbt`
+
+### Recommended `sbt` plugins
+
+[Example `project/plugins.sbt`](https://github.com/guardian/etag-caching/blob/main/project/plugins.sbt)
+
+* [`sbt-version-policy`](https://github.com/scalacenter/sbt-version-policy) - to supply automatic compatibility-based
+  version-numbering that follows [recommended Scala semver rules](https://www.scala-lang.org/blog/2021/02/16/preventing-version-conflicts-with-versionscheme.html#early-semver-and-sbt-version-policy)
+  on your library, as well as automatically setting the [`versionScheme`](https://www.scala-sbt.org/1.x/docs/Publishing.html#Version+scheme)
+  of your project to `"early-semver"` (essential).
+* [`sbt-release`](https://github.com/sbt/sbt-release)
+* [`sbt-sonatype`](https://github.com/xerial/sbt-sonatype) - you currently need to have this in your project's
+  `plugins.sbt`, as the [`🎊 Create artifacts`](https://github.com/guardian/gha-scala-library-release-workflow/blob/7d278d4d44e30b4b4c0f6791053bdeb40b8159cb/.github/workflows/reusable-release.yml#L141-L158)
+  job expects it, but in the future, the workflow will probably supply this automatically.
+
+### Recommended `sbt` settings
```

Comment:
Not an sbt setting, but I suppose it's implicit that any project using this reusable workflow defines the next version using a `version.sbt` file (which isn't currently the case for mobile-apps-api-models). I think mapi models is an anomaly in terms of the release process though, so maybe it's superfluous to mention the necessity for a version.sbt in these docs.

Reply:
Good point - added in 99340564e83d511aa77134f894eed0588a2b5acf